storage/statedb/encoding.go

```diff
@@ -119,3 +123,13 @@ func prefixLen(a, b []byte) int {
 func hasTerm(s []byte) bool {
 	return len(s) > 0 && s[len(s)-1] == 16
 }
+
+// HexKeyPathToHashString turns hex nibbles of trie path key into hash string of the trie path key.
+func HexKeyPathToHashString(path []byte) string {
```

Comment:
Why do you turn `Path` to `Hash`? Isn't it better if we can how the nodes are connected?

Reply:
Path is bytes. Hash is ascii. 
So hash is more short and understandable. :)
This is an example. 
`10101010 -> 0xAAAA`

Reply:
To me, the word `Hash` sounds like you hashed the path.
Could you change the function name to `HexPathToString` or `HexKeyPathToEncodedString`?
```suggestion
// HexPathToString turns hex nibbles of trie path key into string of the trie path key.
func HexPathToString(path []byte) string {
```